tox.ini

```diff
@@ -0,0 +1,10 @@
+[flake8]
+# E741 = Ambiguous variable name
+# C901 = Overcomplex function
+ignore = E741, C901
+exclude = .git,
```

Comment:
Shouldn't we also exclude `.github`, `static` and the other config files in root as well? Like the travis config, gitignore, the procfile, the readme and so on? I haven't used flake8 often before, so maybe I am wrong @ankan17

Reply:
We can ignore those files explicitly but they are only a few files and there won't be any error generated from those files since they are not .py files. If you wish, I can put those in the ignore section. But it's just less code vs less checking. :smiley:

Reply:
Yeah but you may do it just for formality reasons :+1: :smile: